### ISSUE TITLE
How i can add menu block to main block category permanently?

### ISSUE THREAD

**Issue by yudaaditia** [LLK/scratch-gui#6626]
i am use scratch-vm for add extensions to scratch-gui but it is not permanenly, what is the step so i can add menu block my own, to main block category?

**Comment by apple502j** [comment]
You can probably load the extension automatically.

**Comment by yudaaditia** [comment]
> You can probably load the extension automatically.

yes, i think so. but i don't know where i have to place the code, where is the directory to load the menu defaults?

I tried adding the file newBlock in scratch-vm in path node_modules / scratch-block / blocks_vertical and registering it in runtime. but it doesn't appear when I recompile it

**Comment by rmaw22** [comment]
> You can probably load the extension automatically.

can u explain how it works ? thx..
